### ISSUE TITLE
Relative imports are broken in Docker

### ISSUE THREAD

**Issue by paddatrapper** [ClearlyEnergy/helix-label#2]
While trying to run seed-python3 in docker, I get the following error:

```
  File "/seed/src/helix/helix/urls.py", line 2, in <module>                                                                                            
    from helix.views import (          
  File "/seed/src/helix/helix/views.py", line 56, in <module>
    from label import label                
  File "/seed/src/label/label/label.py", line 1, in <module>
    from populate_residential_green_addendum import write_green_addendum_pdf
ModuleNotFoundError: No module named 'populate_residential_green_addendum'
```

**Comment by paddatrapper** [comment]
There should probably be a `__init__.py` file in /label, and then the `.populate_residential_green_addendum` import should work

**Comment by paddatrapper** [comment]
Fixed in 2031f31b429bd65cc95e465927078ec8b09ef999
